### PR TITLE
More Advancements

### PR DESCRIPTION
**Pull request by InfectedGriffon**
i swear if git messes this up again

### CODE REVIEW

File: src/main/java/com/mraof/minestuck/data/MSAdvancementProvider.java [hunk 1]
```diff
@@ -64,24 +73,31 @@ public static DataProvider create(PackOutput output, CompletableFuture<HolderLoo
...
+		Advancement halfwayPoint = Advancement.Builder.advancement().parent(doubleDigits).display(BoondollarsItem.setCount(new ItemStack(MSItems.BOONDOLLARS.get()), 1000), Component.translatable(title(HALFWAY_POINT)), Component.translatable(desc(HALFWAY_POINT)), null, FrameType.TASK, true, true, false).addCriterion("reach_rung", EcheladderTrigger.Instance.rung(MinMaxBounds.Ints.atLeast(25))).save(saver, save_loc(HALFWAY_POINT));
+		Advancement bigOneMil = Advancement.Builder.advancement().parent(halfwayPoint).display(BoondollarsItem.setCount(new ItemStack(MSItems.BOONDOLLARS.get()), 1_000_000), Component.translatable(title(BIG_ONE_MIL)), Component.translatable(desc(BIG_ONE_MIL)), null, FrameType.GOAL, true, true, false).addCriterion("reach_rung", EcheladderTrigger.Instance.rung(MinMaxBounds.Ints.atLeast(44))).save(saver, save_loc(BIG_ONE_MIL));
```
Comment (kirderf1) [inline src/main/java/com/mraof/minestuck/data/MSAdvancementProvider.java:97]: Huh, didn't know you could use underscores in number literals like that. The more you know!

**Review by kirderf1 — APPROVED** [review]

**Comment by kirderf1** [comment]
@Dweblenod do you want to have a look as well?

**Review by Dweblenod — APPROVED** [review]
Looks good! Gave some optional content suggestions but it looks solid

**Comment by Dweblenod** [comment]
Since my comments were optional, just let me know whether you feel up to adding them. I can merge this whenever if you dont want to make changes!

**Comment by InfectedGriffon** [comment]
Sorry I was planning on responding to this, then got wrapped up in something else. I think I will do both of your suggestions quickly though.

**Pull request merged**